exercises/kindergarten-garden/test/Tests.hs

```diff
@@ -47,18 +47,10 @@ specs = describe "kindergarten-garden" $ do
 
       plants "Alice"   `shouldBe` [ Violets , Radishes, Violets , Radishes ]
       plants "Bob"     `shouldBe` [ Clover  , Grass   , Clover  , Clover   ]
-      plants "Charlie" `shouldBe` [ Violets , Violets , Clover  , Grass    ]
```

Comment:
I don't care too much about the track-specific cases. If they are important, we should include them in `x-common`, otherwise, I think we should remove them.